core/src/main/java/nl/nn/adapterframework/doc/model/FrankDocModel.java

```diff
@@ -293,12 +296,16 @@ private void documentAttribute(FrankAttribute attribute, Method method, FrankEle
 						log.trace(String.format("Describing element of attribute [%s].[%s] is [%s]",
 								attributeOwner.getFullName(), attribute.getName(), attribute.getDescribingElement().getFullName()));
 					}
-					attribute.parseIbisDocAnnotation(ibisDoc);
+					if(! attribute.parseIbisDocAnnotation(ibisDoc)) {
+						log.warn(String.format("FrankAttribute [%s] of FrankElement [%s] does not have a configured order", attribute.getName(), attributeOwner.getFullName()));
+					}
 					if(parsed.hasOrder) {
 						attribute.setOrder(parsed.getOrder());
 						if(log.isTraceEnabled()) {
 							log.trace(String.format("Attribute [%s] has order from @IbisDocRef: [%d]", attribute.getName(), attribute.getOrder()));
 						}
+					} else {
+						
```

Comment:
I removed the else clause because it is not needed. The "then" part overrules the order with the order from an @IbisDocRef annotation. If there is no such order, then the @IbisDoc annotation of the referred method is used.